proto/message_contents/private_key.proto

```diff
@@ -12,7 +12,34 @@ import "message_contents/public_key.proto";
 option go_package = "github.com/xmtp/proto/v3/go/message_contents";
 option java_package = "org.xmtp.proto.message.contents";
 
-// PrivateKey generalized to support different key types
+// A private key with a corresponding public key that is provably linked
+// to a user account
```

Comment:
What makes the key account linked?

Reply:
The "Create Identity Signature" is used to Link a blockchain account to an XMTP Account/Identity Key. This is the definition of account linking. The concept behind an accountLinkedKey is to split up how keys are used in the Protocol by intent rather than cryptographic construct. 

E.g The sendKey work will enable a wider range of signatures to be used to link accounts such as SIWE

Reply:
Ah right, I confused myself. SignedPrivateKey isn't actually signed itself, it's the corresponding PublicKey that is signed. It's been a while 😓